### PR TITLE
Add VSCodium v1.27.1

### PR DESCRIPTION
**Pull request by MatiasConTilde**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by adidalal** [comment]
@MatiasConTilde Does this conflict with `vscode`, or can they be installed in harmony? That would also determine what would be appropriate to put in `uninstall`

File: Casks/vscodium.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'vscodium' do
```
Comment (vitorgalvao) [inline Casks/vscodium.rb]: Pinging @stripedpajamas as well.

This one is a problem, as it does not adhere to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md) (which it needs to, as it helps with duplicates). Two options:

1. Rename the cask `code-oss`.
2. Rename the app `VSCodium` *on your end*.

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by adidalal** [comment]
And do they use the same settings folder? (~/.vscode or something IIRC)

**Comment by MatiasConTilde** [comment]
@adidalal No, everything is separte, even the extensions. The config folders are `~/.vscode/`and `~/.vscode-oss/`

@vitorgalvao I personally prefer the second option and opened https://github.com/VSCodium/vscodium/issues/11, but as I'm not a mantainer I can't change it myself. If VSCodium's devs don't agree, I will rename it to `code-oss`.

File: Casks/vscodium.rb (hunk 1 above)
Comment (stripedpajamas) [reply Casks/vscodium.rb]: Changed the name of the app to VSCodium 😄thanks @vitorgalvao and @MatiasConTilde

Comment (vitorgalvao) [reply Casks/vscodium.rb]: Thank you @stripedpajamas. @MatiasConTilde Care to update the cask?

**Review by vitorgalvao — APPROVED** [review]

**Review by stripedpajamas — APPROVED** [review]

**Pull request merged**